cram/cram_structs.h

```diff
@@ -706,7 +706,6 @@ typedef struct cram_fd {
 
     // options
     int decode_md; // Whether to export MD and NM tags
-    int verbose;
```

Comment:
This changes the size of cram_fd struct and moves the elements down a few bytes.  The ```cram_fd``` definition is internal and the external code that accesses it does so via accessor functions `cram_fd_get_fp`, `cram_fd_get_header`, `cram_minor_vers` and `cram_major_vers` in cram_external.c.

So I *think* this is fine for compatibility and doesn't break ABI at all despite the structure size change.